addons/beehave/blackboard.gd

```diff
@@ -7,13 +7,14 @@ const DEFAULT = "default"
 ## multiple nodes of the behavior tree.
 @export var blackboard: Dictionary = {}:
```

Comment:
When exporting dictionaries in Godot, they will be shared by default. So we have to manually duplicate them.